presto-docs/src/main/sphinx/connector/hana.rst

```diff
@@ -0,0 +1,100 @@
+mount the HANA connector as the ``hana`` catalog.
+Create the file with the following contents, replacing the
+connection properties as appropriate for your setup:
+
+.. code-block:: none
+
+    connector.name=hana
+    connection-url=jdbc:sap://[serverName[\instanceName][:portNumber]]
+    connection-user=root
+    connection-password=secret
+
+Multiple HANA Databases or Servers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The HANA connector can only access a single database within
+a HANA server. Thus, if you have multiple HANA databases,
+or want to connect to multiple instances of the HANA, you must configure
+multiple catalogs, one for each instance.
+
+To add another catalog, simply add another properties file to ``etc/catalog``
+with a different name (making sure it ends in ``.properties``). For example,
+if you name the property file ``sales.properties``, Presto will create a
+catalog named ``sales`` using the configured connector.
+
+General Configuration Properties
+---------------------------------
+
+================================================== ==================================================================== ===========
+Property Name                                      Description                                                          Default
+================================================== ==================================================================== ===========
+``user-credential-name``                           Name of the ``extraCredentials`` property whose value is the JDBC
+                                                   driver's user name. See ``extraCredentials`` in `Parameter Reference
+                                                   <https://prestodb.io/docs/current/installation/jdbc.html
+                                                   #parameter-reference>`_.
+
+``password-credential-name``                       Name of the ``extraCredentials`` property whose value is the JDBC
+                                                   driver's user password. See ``extraCredentials`` in `Parameter
+                                                   Reference <https://prestodb.io/docs/current/installation/jdbc.html
+                                                   #parameter-reference>`_.
+
+``case-insensitive-name-matching``                 Match dataset and table names case-insensitively.                    ``false``
+
+``case-insensitive-name-matching.cache-ttl``       Duration for which remote dataset and table names will be
+                                                   cached. Set to ``0ms`` to disable the cache.                         ``1m``
+================================================== ==================================================================== ===========
+
+Querying HANA
+-------------------
+
+The HANA connector provides access to all schemas visible to the specified user in the configured database.
+For the following examples, assume the HANA catalog is ``hana``.
```

Comment:
This is another example of well-written documentation, explaining how to read the examples in this section.